diff -Naur wxWidgets-3.1.2-old/src/osx/cocoa/window.mm wxWidgets-3.1.2/src/osx/cocoa/window.mm
--- wxWidgets-3.1.2-old/src/osx/cocoa/window.mm	2019-08-17 21:49:43.000000000 -0700
+++ wxWidgets-3.1.2/src/osx/cocoa/window.mm	2019-08-17 21:50:54.000000000 -0700
@@ -3151,7 +3151,8 @@
             wxTopLevelWindow* toplevel = wxDynamicCast(peer,wxTopLevelWindow);
 
             if ( toplevel == NULL || toplevel->GetShape().IsEmpty() )
-                [targetView setBackgroundColor: col.OSXGetNSColor()];
+                [targetView setBackgroundColor:
+                        col.IsOk() ? col.OSXGetNSColor() : nil];
         }
     }
 }
